ibtk/src/utilities/LibMeshSystemVectors.cpp

```diff
@@ -0,0 +1,207 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2020 - 2020 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+/////////////////////////////// INCLUDES /////////////////////////////////////
+
+#include <ibtk/FEDataManager.h>
+#include <ibtk/LibMeshSystemVectors.h>
+
+#include <tbox/Utilities.h>
+
+#include <libmesh/equation_systems.h>
+#include <libmesh/petsc_vector.h>
+#include <libmesh/system.h>
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+/////////////////////////////// STATIC ///////////////////////////////////////
+
+namespace
+{
+bool
```

Comment:
Should this be `inline`?
```suggestion
inline bool
```

Reply:
If its in the same translation unit the compiler will consider it for inlining, so I don't think this matters.